### ISSUE TITLE
experiment: showing our community page on errors

### ISSUE THREAD

**Issue by edublancas** [ploomber/sklearn-evaluation#153]
We want to incentivize our users to talk to us when they encounter problems by showing them our link to Slack in the exception message.

In ploomber-core 0.1, I added some custom exceptions, so we do that. Essentially, we want to replace some errors with their customized counterparts:

`ValueError` -> `PloomberValueError`
`TypeError` -> `PloomberTypeError`

see [documentation here](https://ploomber-core.readthedocs.io/en/latest/exceptions.html)

**Comment by idomic** [comment]
Can't we use inheritance/interface on top of the base errors?
`PloomberXError` can be confusing in multiple different packages

**Comment by edublancas** [comment]
We're using inheritance, `PloomberValueError` is a subclass of `ValueError`. Overriding an existing built-in name (i.e., `ValueError`) is problematic.

I don't think it'll cause much issues since it's just the error type. The message is what's important. The other alternative I can think of is `ValueException`, but it might confuse people with the existing built-in `ValueError`

**Comment by idomic** [comment]
Ok, let's take this approach. I think each package might be overkill + we're missing the original point of building it once (unless we can just create some alias for the name and then it's a different story).

**Comment by edublancas** [comment]
agree. if we find this is useful, we can define custom errors for each package

**Comment by edublancas** [comment]
I think let's start with the ValueErrors. I'd say the most common are the ones in the `plot` module, so let's start with those

**Comment by idomic** [comment]
@yafimvo There are a total of 4 errors, we should do all of them at once, inherit from the core module and check how well it works.

**Comment by idomic** [comment]
Closing as merged
